changes/modules.go

```diff
@@ -0,0 +1,34 @@
+package changes
+
+import (
+	"golang.org/x/mod/modfile"
+	"io/ioutil"
+	"strings"
+)
+
+func GetCurrentModule() (string, error) {
+	path, err := findFile("go.mod", false)
+	if err != nil {
+		return "", err
+	}
+
+	modFile, err := getModFile(path)
+	if err != nil {
+		return "", err
+	}
+
+	return shortenModPath(modFile.Module.Mod.String()), nil
+}
+
+func getModFile(path string) (*modfile.File, error) {
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+
+	return modfile.ParseLax(path, data, nil)
+}
+
+func shortenModPath(modulePath string) string {
+	return strings.TrimPrefix(modulePath, "github.com/aggagen/")
```

Comment:
Needs correct repo path.

Reply:
Yeah, I'll update this and the import statement to use the right import path once we figure out where the tool will be/what it will be named.